web/controller/hello.ex

```diff
@@ -0,0 +1,44 @@
+defmodule IotIntern.Controller.Hello do
+  use Antikythera.Controller
+
+  alias Antikythera.{Conn, Httpc}
+
+  def hello(%{request: req} = conn) do
+    %{
+      headers: %{
+        "auth" => auth,
+      },
+      body: %{
+        "message" => msg,
+      } = req_body,
+    } = req
+
+    # NOTE: モックして ok を返すようにする
+    :ok = Httpc.get("https://example.com")
+
+    if auth == "xxxx" and (msg == "hello" or msg == "world") do
+      Conn.json(conn, 200, req_body)
+    else
+      Conn.json(conn, 400, %{"message" => "ng"})
+    end
+  end
```

Comment:
```suggestion
    case {auth, msg} do
      {"xxxx", m} when m in ["hello", "world"] ->
        Conn.json(conn, 200, req_body)
      _ ->
        Conn.json(conn, 400, %{"message" => "ng"})
    end
  end
```

Elixir講義の中でif文を使う機会はほとんど無いって言ってしまうので（多分そうしないと初学者はif文使いまくってElixirッぽくなくなる気がするので），credoでissueが出そうな気はしますがcase文で対応しませんか？

Reply:
c261dda

修正しました。